### PR TITLE
pybind/rados: add WriteOp::execute()

### PR DESCRIPTION
**Pull request by zjcmszh**
Signed-off-by: Zhang Jiao <zhangjiao@cmss.chinamobile.com>

### CODE REVIEW

File: src/test/pybind/test_rados.py [hunk 1]
```diff
@@ -528,6 +528,13 @@ def test_write_ops(self):
             with assert_raises(ObjectNotFound):
                 self.ioctx.read('write_ops')
 
+    def test_execute_op(self):
+        write_op = self.ioctx.create_write_op()
+        write_op.write_op_exec("hello", "record_hello", "ebs")
+        comp = self.ioctx.operate_write_op(write_op, "object")
+        self.ioctx.release_write_op(write_op)
+        eq(self.ioctx.read('object'), b"Hello, ebs!")
```
Comment (tchaikov) [inline src/test/pybind/test_rados.py]: could you use `WriteOpCtx` instead? like
```py
with WriteOpCtx() as write_op:
  write_op.exec("hello", "record_hello", "ebs")
  self.ioctx.operate_write_op(write_op, "object")
eq(self.ioctx.read('object'), b"Hello, ebs!")
```
 also, could you remind me where `write_op_exec()` is defined?

**Comment by zjcmszh** [comment]
@tchaikov   Sorry, i changed the function name, but forgot to change it in the test code.

**Comment by tchaikov** [comment]
@zjcmszh thanks for explaining. but since you have test in your change, i'd suggest at least run it before pushing it.

**Comment by zjcmszh** [comment]
Thanks your advices, I will be more careful and this mistake shall not happen again.

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2019-11-17_01:29:22-rados-wip-kefu-testing-2019-11-17-0135-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2019-11-19_00:33:33-rados-wip-kefu-testing-2019-11-17-0135-distro-basic-mira/

test failures are either tracked or fixed by

- https://tracker.ceph.com/issues/42876
- #31689

**Comment by zjcmszh** [comment]
@tchaikov  I see, thanks a lot.

**Pull request merged**